templates/react/utils/itemToLinks.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {API_PATH} from '../api/_entrypoint';
+
+export default function (itemArray) {
+  if (Array.isArray(itemArray)) {
+    return (
```

Comment:
Can't it be a oneliner?

`return  Array.isArray(items) ? items.map(item => createLink(item)) : createLink(items);`